<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify_Light</class>
 <widget class="QMainWindow" name="Modify_Light">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="Light_ON_OFF">
       <property name="text">
        <string>Light ON/OFF</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>150</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_EDIT">
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="Ambient_colour">
         <property name="text">
          <string>Ambient Colour</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Diffuse_colour">
         <property name="text">
          <string>Diffuse Colour</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Specular_colour">
         <property name="text">
          <string>Specural Colour</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>     Light Origingation Point</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string> X-Axis</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="X_Axis_Origin">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string> Y-Axis</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="Y_Axis_Origin">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string> Z-Axis</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="Z_Axis_Origin">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>    Light Focussing Point </string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string> X-Axis</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="X_Axis_Focussing">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string> Y-Axis</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="Y_Axis_Focussing">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string> Z-Axis</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="Z_Axis_Focussing">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>481</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>        Edit Apex angle of Light</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QSlider" name="Apex_angle_Light_slider">
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>       Edit Intensity of Light</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QSlider" name="Intensity_Of_Light_slider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
